### PR TITLE
Dp branch

### PR DESCRIPTION
**Pull request by dirkphilip**

### CODE REVIEW

File: csemlib/utils.py [hunk 1]
```diff
@@ -30,15 +30,17 @@ def cart2sph(x, y, z):
...
+    r = np.sqrt(x ** 2 + y ** 2 + z ** 2)
+
     # Break obviously if radius is zero. In that case we're just
     # at the center of the Earth.
-    try:
-        c = np.math.acos(z / r)
-    except ZeroDivisionError:
-        return 0, 0, 0
-    l = np.math.atan2(y, x)
+    with np.errstate(invalid='ignore'):
```
Comment (michael-afanasiev) [inline csemlib/utils.py:37]: Can you add a comment explaining what this does?

**Review by michael-afanasiev — CHANGES_REQUESTED** [review]
Sweet stuff

File: csemlib/utils.py (hunk 1 above)
Comment (dirkphilip) [reply csemlib/utils.py:37]: This handles floating point errors such as dividing by zero at the core. 
Dividing by zero in this function will now return a nan without warning, which is later replaced by zero

Comment (michael-afanasiev) [reply csemlib/utils.py:37]: Hey man. More specifically I meant a comment in the code to say what's going on there. Like

`# handle division by 0 at core`

**Review by michael-afanasiev — APPROVED** [review]

**Pull request merged**